### PR TITLE
added link of fontastic

### PR DESCRIPTION
**Pull request by lastresmoya**

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "assert": "^2.0.0",
+    "bn.js": "^5.0.0",
```
Comment (chaitanyapotti) [inline app/package.json]: why is this added?

File: app/public/index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
     <meta name="keywords" content="crypto, wallet, oauth, login, best, ethereum, torus">
     <meta name="author" content="Torus Labs">
     <link rel="shortcut icon" href="<%= BASE_URL %>favicon.png" />
+    <link href="https://file.myfontastic.com/upUhdYWuxL9ArmsPAbofL6/icons.css" rel="stylesheet"
```
Comment (chaitanyapotti) [inline app/public/index.html]: we are already using a lot of icons. please download the required icon and put it into the public folder and use it in the app

**Comment by tetratorus** [comment]
I think as Chai has pointed out it's not necessary to import the icons in this way, it'll increase the time required for a dapp to load our wallet
